cpp/include/cudf/datetime.hpp

```diff
@@ -133,6 +133,57 @@ std::unique_ptr<cudf::column> extract_second(
   cudf::column_view const& column,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief  Extracts millisecond fraction from any date time type and returns an int16_t
+ * cudf::column.
+ *
+ * A millisecond fraction is only the 3 digits that make up the millisecond portion of a duration.
+ * For example, the microsecond fraction of 1.234567890 seconds is 234.
```

Comment:
Microsecond?

Reply:
Thanks for the catch @ttnghia ! Fixed it.